Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Resources/OutputResources/config.json

```diff
@@ -9,5 +9,8 @@
     "Port": "8086",
     "Username": "ubuntu",
     "ApiKey": "N8VHmvjqq2SnYLR4R_BbS6ggM2cCDyvrID6XZqYUJ-0DMDQUxLDHiwVGYNtJAhqnxupX4X7TZwNYkOUc1FIX8A=="
+  },
+  "PythonScript": {
+    "FileName": "python2.py"

```

Comment:
The script pushed is called `python1.py`, did you forget to push `python2.py`?

Reply:
Ohh i was testing it, if it also fails with the wrong file 😂. I will fix it now